src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -23,28 +33,73 @@
     public static void main(String[] args) {
         Vertx vertx = Vertx.vertx();
 
-        AmqpBridgeConfig amqpBridgeConfig = AmqpBridgeConfig.fromMap(System.getenv());
-        if (amqpBridgeConfig.getEndpointConfig().isEnabled()) {
-            AmqpBridge amqpBridge = new AmqpBridge(amqpBridgeConfig);
-            vertx.deployVerticle(amqpBridge, done -> {
-                if (done.succeeded()) {
-                    log.debug("AMQP verticle instance deployed [{}]", done.result());
-                } else {
-                    log.debug("Failed to deploy AMQP verticle instance", done.cause());
-                }
-            });
-        }
+        ConfigStoreOptions fileStore = new ConfigStoreOptions()
+                .setType("file")
+                .setOptional(true)
+                .setFormat("properties")
+                .setConfig(new JsonObject().put("path", getFilePath(args[0])));
+
+        ConfigRetrieverOptions options = new ConfigRetrieverOptions().addStore(fileStore);
+
+        ConfigRetriever retriever = ConfigRetriever.create(vertx, options);
+        retriever.getConfig(ar -> {
+            AmqpConfig amqpConfig = new AmqpConfig(ar.result().getBoolean("amqp.enabled", AmqpConfig.DEFAULT_AMQP_ENABLED),
+                    AmqpMode.from(ar.result().getString("amqp.mode", AmqpConfig.DEFAULT_AMQP_MODE)),
+                    ar.result().getInteger("amqp.flowCredit", AmqpConfig.DEFAULT_FLOW_CREDIT),
+                    ar.result().getString("amqp.host", AmqpConfig.DEFAULT_HOST),
+                    ar.result().getInteger("amqp.port", AmqpConfig.DEFAULT_PORT),
+                    ar.result().getString("amqp.messageConverter", AmqpConfig.DEFAULT_MESSAGE_CONVERTER),
+                    ar.result().getString("amqp.certDir", AmqpConfig.DEFAULT_CERT_DIR));
```

Comment:
If we are going to remove the env var, maybe you could avoid this hard-coded properties and use the related fields in `AmqpConfig` class which today are env vars but changing to these properties.
I.e. 

`private static final String AMQP_ENABLED = "AMQP_ENABLED";`

in

`public static final String AMQP_ENABLED = "amqp.enabled";`